scripts/grt/export.py

```diff
@@ -155,16 +156,15 @@ def annotate(label, human_label=None):
                 continue
 
             try:
-                handle_job.write(str(job[0]))  # id
-                handle_job.write('\t')
-                handle_job.write(job[2])  # tool_id
-                handle_job.write('\t')
-                handle_job.write(job[3])  # tool_version
-                handle_job.write('\t')
-                handle_job.write(job[4])  # state
-                handle_job.write('\t')
-                handle_job.write(str(job[5]))  # create_time
-                handle_job.write('\n')
+                line = [
+                    str(job[0]), # id
+                    job[2], # tool_id
+                    job[3], # tool_version
+                    job[4], # state
+                    str(job[5]) # create_time
+                ]
+                cline = unicodify('\t'.join(line) + '\n')
+                handle_job.write(cline.encode('utf-8'))
```

Comment:
The `.encode('utf-8')` is not needed if you open `handle_job` with:
```python
import io
...
    handle_job = io.open(REPORT_BASE + '.jobs.tsv', 'w', encoding='utf-8')
```

Reply:
Huh, neat. Can do!